composer.json

```diff
@@ -7,7 +7,7 @@
         "php": ">=5.4.0"
     },
     "require-dev": {
-        "phpunit/phpunit": "^4|^5",
+        "phpunit/phpunit": "^4.8|^5.7",
```

Comment:
Because that's an unrelated change (See also https://github.com/async-interop/promise/pull/43#issuecomment-274419903). You can still change the versions in a separate PR.